app/scripts/services/tile-proxy.js

```diff
@@ -83,19 +84,28 @@ const debounce = (func, wait) => {
     requestMapper = {};
   };
 
-  const debounced = (request, ...args) => {
-    bundleRequests(request);
+  const callFunc = (request, ...args) => {
```

Comment:
Can we get a comment describing what this is for?

Reply:
Oh, I see that it's used in debounced. It would still be helpful to have a short comment to that effect.

Reply:
Sure will do!

Reply:
Done!